### PR TITLE
Updateadd

### PR DESCRIPTION
**Pull request by cehsu**
uncommented additional routers, removed restaurant from routes.js, now getting the following error: "TypeError: Cannot read property 'authenticate' of undefined" any theories?

### CODE REVIEW

**Comment by CodeRISHI** [comment]
The error I think occurs because you need to pass the passport module to `routes.js` in `server.js`

So, you would need to add:

`require('./config/routes.js')(app, passport)` in `server.js`

It doesn't recognize `passport` in `routes.js` right now because it can't see it.

**Comment by cehsu** [comment]
Hmmm, i think it is there at the bottom?
line 73: require('./config/routes.js')(app, express, passport);

**Comment by jreidgreer** [comment]
Hey @cehsu the `require('./config/routes.js')(app, express, passport);` _must_ remain at the bottom of the file. In addition, the other routing should probably be moved to the bottom as well for when we secure those endpoints. Using passport and sessions as your middleware requires you to be careful about the order of the files.

**Comment by jreidgreer** [comment]
In addition we now have two `require('./config/routes.js')(app, express, passport);`, so please remove the addition, move the other routing down to the bottom, and recommit. Thanks!

File: server/server.js [hunk 1]
```diff
@@ -16,9 +17,9 @@ require('./config/middleware.js')(app, express);
 
 
 
-// app.use('/api/restaurants', restaurantRouter);
-// app.use('/api/users', userRouter);
-// app.use('/api/likes', likesRouter);
+app.use('/api/restaurants', restaurantRouter);
+app.use('/api/users', userRouter);
+app.use('/api/likes', likesRouter);
```
Comment (jreidgreer) [inline server/server.js]: @cehsu, can you move these to the bottom after the `routes.js`?

File: server/server.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ var session = require('express-session');
 var passport = require('passport');
 var GitHubStrategy = require('passport-github').Strategy;
 var User = require('./users/userModel.js');
+// var Restaurant = require('./restaurants/restaurantModel.js');
```
Comment (jreidgreer) [inline server/server.js]: @cehsu What's the purpose of this model here?

**Pull request closed without merging**